### DISCUSSION TITLE
v3 errors (tailwind and field-wrapper)

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#7348] [Help]
hi, while upgrading the latest version i get the following errors:

Unable to locate a class or view for component [forms::field-wrapper].

and one related to tailwind while i try to run npm run build:

[vite:css] Failed to find '../../vendor/filament/filament/resources/css/app.css'
  in [
    C:\laragon\www\modatex-chile-tres\resources\css
  ]
file: C:\laragon\www\modatex-chile-tres\resources\css\filament.css
error during build:
Error: Failed to find '../../vendor/filament/filament/resources/css/app.css'
  in [
    C:\laragon\www\modatex-chile-tres\resources\css
  ]

**Comment by pboivin** [comment]
Did you run `php artisan view:clear` as part of the upgrade?
